testapp/src/main/res/values/strings.xml

```diff
@@ -117,4 +117,5 @@
     <string name="deeplink_activity_input_hint">Input here</string>
     <string name="deeplink_activity_no_deeplink">No deeplink available!</string>
     <string name="deeplink_activity_not_saved">No deeplink has been saved yet.</string>
+    <string name="miniapp_by_input">MiniApp By Input</string>
```

Comment:
Do we have this change done in iOS as well? @laubengaier

Reply:
actually this should be called `URL` I think @munir-rakuten already updated the name? 🤔

Reply:
I see, I made an update, please check